lib/matrix.gi

```diff
@@ -370,8 +370,8 @@ end );
 ##
 BindGlobal( "Matrix_OrderPolynomialInner", function( fld, mat, vec, vecs)
     local d, w, p, one, zero, zeroes, piv,  pols, x, t;
-    Info(InfoMatrix,2,"Order Polynomial Inner on ",Length(mat[1]),
-         " x ",Length(mat)," matrix over ",fld," with ",
+    Info(InfoMatrix,2,"Order Polynomial Inner on ",NrCols(mat),
```

Comment:
As far as I understand, the matrix must be square,
but still it is irritating that the message talks about a `NCols x NRows matrix`.